File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/cpls/CPL3010U00GrdGumListItemInfo.java
package nta.med.data.model.ihis.cpls;

public class CPL3010U00GrdGumListItemInfo {
	private String specimenSer;
    private Double fkcpl2010;
    private String gumsaName;
    private String jangbiName;
    private String cplResult;
    private String specimenName;
    private String hangmogCode;
    private String partJubsuja;
    private String subJubsuYn;
    private String spcialName;
    private String fkocs;
    private String confirmYn;
	public CPL3010U00GrdGumListItemInfo(String specimenSer, Double fkcpl2010,
			String gumsaName, String jangbiName, String cplResult,
			String specimenName, String hangmogCode, String partJubsuja,
			String subJubsuYn, String spcialName, String fkocs, String confirmYn) {
		super();
		this.specimenSer = specimenSer;
		this.fkcpl2010 = fkcpl2010;
		this.gumsaName = gumsaName;
		this.jangbiName = jangbiName;
		this.cplResult = cplResult;
		this.specimenName = specimenName;
		this.hangmogCode = hangmogCode;
		this.partJubsuja = partJubsuja;
		this.subJubsuYn = subJubsuYn;
		this.spcialName = spcialName;
		this.fkocs = fkocs;
		this.confirmYn = confirmYn;
	}
	public String getSpecimenSer() {
		return specimenSer;
	}
	public void setSpecimenSer(String specimenSer) {
		this.specimenSer = specimenSer;
	}
	public Double getFkcpl2010() {
		return fkcpl2010;
	}
	public void setFkcpl2010(Double fkcpl2010) {
		this.fkcpl2010 = fkcpl2010;
	}
	public String getGumsaName() {
		return gumsaName;
	}
	public void setGumsaName(String gumsaName) {
		this.gumsaName = gumsaName;
	}
	public String getJangbiName() {
		return jangbiName;
	}
	public void setJangbiName(String jangbiName) {
		this.jangbiName = jangbiName;
	}
	public String getCplResult() {
		return cplResult;
	}
	public void setCplResult(String cplResult) {
		this.cplResult = cplResult;
	}
	public String getSpecimenName() {
		return specimenName;
	}
	public void setSpecimenName(String specimenName) {
		this.specimenName = specimenName;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getPartJubsuja() {
		return partJubsuja;
	}
	public void setPartJubsuja(String partJubsuja) {
		this.partJubsuja = partJubsuja;
	}
	public String getSubJubsuYn() {
		return subJubsuYn;
	}
	public void setSubJubsuYn(String subJubsuYn) {
		this.subJubsuYn = subJubsuYn;
	}
	public String getSpcialName() {
		return spcialName;
	}
	public void setSpcialName(String spcialName) {
		this.spcialName = spcialName;
	}
	public String getFkocs() {
		return fkocs;
	}
	public void setFkocs(String fkocs) {
		this.fkocs = fkocs;
	}
	public String getConfirmYn() {
		return confirmYn;
	}
	public void setConfirmYn(String confirmYn) {
		this.confirmYn = confirmYn;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Cpls/CPL3010U00GrdGumListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Cpls
{
    [Serializable]
	public class CPL3010U00GrdGumListItemInfo
	{
		private String _specimenSer;
		private String _fkcpl2010;
		private String _gumsaName;
		private String _jangbiName;
		private String _cplResult;
		private String _specimenName;
		private String _hangmogCode;
		private String _partJubsuja;
		private String _subJubsuYn;
		private String _spcialName;
		private String _fkocs;
		private String _confirmYn;
		private String _dataRowState;

		public String SpecimenSer
		{
			get { return this._specimenSer; }
			set { this._specimenSer = value; }
		}

		public String Fkcpl2010
		{
			get { return this._fkcpl2010; }
			set { this._fkcpl2010 = value; }
		}

		public String GumsaName
		{
			get { return this._gumsaName; }
			set { this._gumsaName = value; }
		}

		public String JangbiName
		{
			get { return this._jangbiName; }
			set { this._jangbiName = value; }
		}

		public String CplResult
		{
			get { return this._cplResult; }
			set { this._cplResult = value; }
		}

		public String SpecimenName
		{
			get { return this._specimenName; }
			set { this._specimenName = value; }
		}

		public String HangmogCode
		{
			get { return this._hangmogCode; }
			set { this._hangmogCode = value; }
		}

		public String PartJubsuja
		{
			get { return this._partJubsuja; }
			set { this._partJubsuja = value; }
		}

		public String SubJubsuYn
		{
			get { return this._subJubsuYn; }
			set { this._subJubsuYn = value; }
		}

		public String SpcialName
		{
			get { return this._spcialName; }
			set { this._spcialName = value; }
		}

		public String Fkocs
		{
			get { return this._fkocs; }
			set { this._fkocs = value; }
		}

		public String ConfirmYn
		{
			get { return this._confirmYn; }
			set { this._confirmYn = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public CPL3010U00GrdGumListItemInfo() { }

		public CPL3010U00GrdGumListItemInfo(String specimenSer, String fkcpl2010, String gumsaName, String jangbiName, String cplResult, String specimenName, String hangmogCode, String partJubsuja, String subJubsuYn, String spcialName, String fkocs, String confirmYn, String dataRowState)
		{
			this._specimenSer = specimenSer;
			this._fkcpl2010 = fkcpl2010;
			this._gumsaName = gumsaName;
			this._jangbiName = jangbiName;
			this._cplResult = cplResult;
			this._specimenName = specimenName;
			this._hangmogCode = hangmogCode;
			this._partJubsuja = partJubsuja;
			this._subJubsuYn = subJubsuYn;
			this._spcialName = spcialName;
			this._fkocs = fkocs;
			this._confirmYn = confirmYn;
			this._dataRowState = dataRowState;
		}

	}
}